bindings/python/pinocchio/display/__init__.py

```diff
@@ -0,0 +1,4 @@
+
+from abstract_display import AbstractDisplay
```

Comment:
I think you should use`from .abstract_display import AbstractDisplay` for python 3, and the same on the next lines.

Reply:
You are totally right. I'll fix this.